docs/inline_icons.rst

```diff
@@ -0,0 +1,85 @@
+.. |semi-colon| replace:: semi-colon (``;``)
+.. |comma| replace:: comma (``,``)
+
+Inline icons
+============
+
+This theme has a builtin role to use icons within lines of text.
+
+.. warning::
+    Using icons in the table of contents or navigation menu is not supported.
+
+.. rst:role:: md-icon
+
+    Set the content of this role to a relative path of an SVG image (excluding the ``.svg`` file
+    extension). This path shall be relative to
+    
+    - the ``.icons`` directory bundled with this theme (this takes precedence)
+    - the documentation's static path(s) (set with `html_static_path` in conf.py)
+
+    Optionally, a CSS class can be specified after the icon's path, but the path and CSS class
+    **must** be separated by |semi-colon|. Do not use spaces. If specifying multiple CSS
+    classes, then separate them with a |comma|.
+
+
+Using inline icons
+------------------
+
+The following sets of icons are bundled with this theme:
+
+.. rst-example:: Using a builtin icon
+
+    - :md-icon:`material/material-design` `Material Design <https://materialdesignicons.com/>`_
+    - :md-icon:`fontawesome/regular/font-awesome` `Font Awesome <https://fontawesome.com/search?m=free>`_
+    - :md-icon:`octicons/mark-github-16` `Octicons <https://octicons.github.com/>`_
+    - :md-icon:`simple/simpleicons` `Simple Icons <https://simpleicons.org/>`_
+
+You can also browse these icons locally from the sphinx-immaterial theme's install path under
```

Comment:
I suppose we could integrate the "icon" index from mkdocs-material into our docs...

Reply:
Yeah, I thought about that too. This is something that can be addressed in the next merge from upstream, assuming we currently exclude the part of the build script that creates the json.